webdriver/client.py

```diff
@@ -368,23 +372,20 @@ def send_command(self, method, url, body=None, key=None):
         if self.session_id is None:
             raise error.SessionNotCreatedException()
 
-        response = self.send_raw_command(method, url, body)
+        url = urlparse.urljoin("session/%s/" % self.session_id, url)
 
-        if response.status != 200:
-            cls = error.get(response.body["value"].get("error"))
-            raise cls(response.body["value"].get("message"))
+        response = self.send_raw_command(method, url, body)
 
-        if key is not None:
-            response.body = response.body[key]
-        if not response.body:
-            response.body = None
+        rv = response.body["value"]
+        if not rv:
+            rv = None
```

Comment:
Yes, this is definitely a bug in wdclient, but this patch doesn’t regress existing behaviour. I suggest we address this in a follow-up patch.